pkgs/applications/science/logic/z3/default.nix

```diff
@@ -1,6 +1,7 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, fetchpatch
```

Comment:
This is unused as it seems.

Reply:
Yes it is! That was from a prior iteration where I had included 4.6 and 4.5, both of which needed a patch to build. I'll remove it.